Towny/src/main/java/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1611,7 +1661,12 @@ public void parseAdminNationCommand(CommandSender sender, String[] split) throws
 			break;
 		case "set":
 			checkPermOrThrow(sender, PermissionNodes.TOWNY_COMMAND_TOWNYADMIN_NATION_SET.getNode());
-			NationCommand.nationSet(sender, StringMgmt.remArgs(split, 2), true, nation);
+			if (split.length == 2) {
+				// Empty Case: /townyadmin nation [nation] set
+				NationCommand.nationSet(sender, StringMgmt.remArgs(split, 2), true, nation);
+			} else {

```

Comment:
Same as with the town version, 0 length args should be handled in parseAdminNationSet with a new HelpMenu